php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S1176.html

```diff
@@ -0,0 +1,38 @@
+<p>
```

Comment:
@nilsmahlstaedt Remove the title from this file, it's not required. Use code samples from [RSPEC](https://jira.sonarsource.com/browse/RSPEC-2045)